### PR TITLE
fix: list should be user-only

### PR DESCRIPTION
**Pull request by userquin**
This PR includes 
- `user-only` on navigation 
- `auth` middleware in lists and list pages
- spanish translation for new entries

### CODE REVIEW

**Review by patak-dev — APPROVED** [review]

File: locales/en.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
     "delete_list": {
       "cancel": "Cancel",
       "confirm": "Delete",
-      "title": "Are you sure you want to delete this list?"
+      "title": "Are you sure you want to delete \"{0}\" list?"
```
Comment (boehs) [inline locales/en.json:122]:
```suggestion
      "title": "Are you sure you want to delete the \"{0}\" list?"
```

**Review by boehs — COMMENTED** [review]
too late

File: locales/en.json (hunk 1 above)
Comment (patak-dev) [reply locales/en.json:122]: Ups, sorry, please send a new PR 🙏🏼

Comment (userquin) [reply locales/en.json:122]: @boehs send a PR to fix it please :pray: , can you include that change in your current PR?

**Pull request merged**